src/components/dialogs/BirthDateSettings.tsx

```diff
@@ -102,10 +114,32 @@ function BirthdayInner({
           onChangeDate={newDate => setDate(new Date(newDate))}
           label={_(msg`Birthday`)}
           accessibilityHint={_(msg`Enter your birth date`)}
-          maximumDate={getDateAgo(13)}
         />
       </View>
 
+      {isUnder18 && (
```

Comment:
Didn't think about this until I tested it, but I think if the user already has set their bday to be <18, we don't need to show the admonition. But if they edit the date to be <18, then we should show it. Tested this locally 👍 
```suggestion
      {isUnder18 && hasChanged && (
```

Reply:
yeah, i agree. totally makes sense.